Documentation/Packaging/MRTK_Modularization.md

```diff
@@ -0,0 +1,167 @@
+# Mixed Reality Toolkit Componentization
+
+One of the great new features of Mixed Reality Toolkit v2 is improved componentization. Wherever possible, 
+individual components are isolated from all but the core layer << >> of the foundation.
+
+The following sections describe the status of the componentization work for each major milestone:
+
+- [Release Candidate 1](#release-candidate-1-status)
+- [Release Candidate 2](#release-candidate-2-plan)
+- [Official Release](#official-release-plan)
+
+> Some features may, by design, take dependencies on MRTK or other components (ex: Unity Plug-ins). 
+The package management tools will ensure that the correct dependencies are imported when one of these 
+features is selected.
```

Comment:
I would suggest not mentioning this until we release the package management tools that ensure this.

Reply:
Otherwise someone's natural question is "what package management tools?" and the answer is "coming soon".

---
In reply to: [273183645](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3874#discussion_r273183645) [](ancestors = 273183645)